lib/members.rb

```diff
@@ -1,17 +1,22 @@
 require 'open-uri'
 require 'open-uri/cached'
 require 'json'
-require_relative 'member_formatter'
+require 'field_serializer'
+require_relative 'member'
 
 OpenURI::Cache.cache_path = '.cache'
 
 class Members
+  include FieldSerializer
+
   def initialize(url)
     @url = url
   end
 
-  def to_a
-    format(all_members)
+  field :members do
+    lower_house_members.map do |mem|
+      Member.new(mem).to_h
```

Comment:
It seems a little odd to coerce these to Hashes here, rather than just returning the underlying objects.